test/boot_arch_test.rb

```diff
@@ -138,7 +138,11 @@ def stub_arch(arch)
         # just to test that it do not add product features
         allow(Yast::ProductFeatures).to receive(:GetStringFeature).and_return("console=ttyS1")
 
-        expect(subject.DefaultKernelParams("/dev/sda2")).to eq "console=ttyS0 splash=verbose"
+        expect(subject.DefaultKernelParams("/dev/sda2")).to eq "console=ttyS0 resume=/dev/sda2 console=ttyS1 splash=silent quiet showopts"
+      end
+
+      it "adds splash=silent quit showopts parameters" do
+        expect(subject.DefaultKernelParams("/dev/sda2")).to include(" splash=silent quiet showopts")
```

Comment:
this is now wrong as it is not other archs context, but ppc arch context. Otherwise it looks good.